chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -742,6 +749,7 @@ public synchronized void loadChunks(TaskTracker progress, World world,
       palette = new BlockPalette();
       worldOctree = new Octree(octreeImplementation, requiredDepth);
       waterOctree = new Octree(octreeImplementation, requiredDepth);
+      emitterGrid = new Grid(requiredDepth, 10); // TODO Make configurable
```

Comment:
Is it an advanced feature that we should add a UI field for in the future or is this more advanced (like e.g. map loader thread count)? If so, maybe add a system property for it (like `chunky.emitterGridCellSize`)?

Reply:
Changing it can allow sampling emitters more far away so it changes the way the render looks (at the cost of performance and maybe memory usage)